### ISSUE TITLE
Why so many `.cursor-tutor-*` folders in my home folder?

### ISSUE THREAD

**Issue by in03** [getcursor/cursor#1454]
## 1) If you can, please include a screenshot of your problem

![image](https://github.com/getcursor/cursor/assets/13340723/4813c0cb-45f4-4a0f-b410-195df3099127)

## 2) Please include the name of your operating system

Windows 10

## 3) If you can, steps to reproduce are super helpful

Install cursor

---

Bit of a nasty surprise to find **47** tutorial folders in my home folder.
Is there any reason why these can't all go in the .cursor folder in the first place?
Assuming this is a bug?

**Comment by truell20** [comment]
Ack! Thank you for reporting this. Just to confirm, have you installed Cursor several times?

**Comment by zzecool** [comment]
> Ack! Thank you for reporting this. Just to confirm, have you installed Cursor several times?

I have the same problem and havent installed cursor many times.

**Comment by in03** [comment]
Nope! First time installing. Just the one installation and it seems to be
working fine.

**Comment by HarryAdney** [comment]
Got these too. Windows 10 pro, installed once.

Any updates?

**Comment by truell20** [comment]
@HarryAdney what version of Cursor did you have when you installed? We pushed a fix for this a few versions back.

**Comment by HarryAdney** [comment]
I can't remember, mate. I've uninstalled it now; sticking with vscode.

**Comment by byroncollins** [comment]
Same issue but I've only just noticed after updating Cursor numerous times so don't know what the original version was. 

Can the .cursor-tutor-* directories be safely deleted?

**Comment by atreya2011** [comment]
On a related note. I close the tutor tab by accident. Is there any way to open it again?

**Comment by jackynkc** [comment]
> @HarryAdney what version of Cursor did you have when you installed? We pushed a fix for this a few versions back.

I also have this issue. I'm using ver 0.40.4

**Comment by cs-mshah** [comment]
I have several of these folders as well in the latest cursor version.
